apps/netbox/ptlsbox/netbox-postgresql.enc.yaml

```diff
@@ -1,24 +1,25 @@
 apiVersion: v1
 data:
```

Comment:
you may need to lint these two files https://github.com/hmcts/cnp-flux-config#sops